File: DsDemo/DemoNet/DemoNetLib/Service/DemoService.cs
using System;

using Dswork.Core.Db;

using DemoNet.Dao;
using DemoNet.Model;

namespace DemoNet.Service
{
    public class DemoService : BaseService<Demo, long>
    {
        public virtual DemoDao demoDao { set; protected get; }

        protected override EntityDao<Demo, long> GetEntityDao()
        {
            return demoDao;
        }
    }
}

File: DsDemo/DemoJava/src/testwork/service/demo/DemoService.java
/**
 * MyBatis样例Service
 */
package testwork.service.demo;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import dswork.core.db.EntityDao;
import dswork.core.db.BaseService;
import testwork.model.Demo;
import testwork.dao.DemoDao;

@Service
@SuppressWarnings("all")
public class DemoService extends BaseService<Demo, Long>
{
	@Autowired
	private DemoDao dao;

	@Override
	protected EntityDao getEntityDao()
	{
		return dao;
	}
}
